### DISCUSSION TITLE
I HAVE 11.6 STUCK ON THE WORMHOLE.. HELP!

### DISCUSSION THREAD

**Discussion by elpochi171** [wormhole-foundation/wormhole#377] [Problems / bugs] [answered]
https://explorer.solana.com/tx/2BRAi7vr4TG5NjJD7v2fgmoyYsnhaBC67hK18SPbkiQGhhD9bN5fZ2CUqbjFiit7VaLHRj9LJs1kHFEYjtgJw3Ek

I am seeing the above TX but my I don't see the WETH on my Phantom wallet. 

Here is the TX on the ETH side. 
https://etherscan.io/tx/0x20d8ec809e23ff7490189f2a096f3443d3aaa218e4c08f0399bc4da6c91ec4b1

It's been more then an Hour... what can I do?

**Comment by leoluk** [comment, marked as answer]
The transfer had completed successfully: https://explorer.solana.com/tx/5XZk1z9Zaa2m5fQGV1YA3uWigjsS4RSHBEUcrfUSELLZEZfUXWs3Q8cEui7UNp43MnZ4QzT5YTkWSXt55HeGXkUZ
